2022-02-18 11:48:35,834 INFO    MainThread:11342 [wandb_setup.py:_flush():71] setting env: {}
2022-02-18 11:48:35,835 INFO    MainThread:11342 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-18 11:48:35,835 INFO    MainThread:11342 [wandb_init.py:_log_setup():371] Logging user logs to /home/simon/CDE_UBS/thesis/testing/wandb/run-20220218_114835-2m3ur0vd/logs/debug.log
2022-02-18 11:48:35,835 INFO    MainThread:11342 [wandb_init.py:_log_setup():372] Logging internal logs to /home/simon/CDE_UBS/thesis/testing/wandb/run-20220218_114835-2m3ur0vd/logs/debug-internal.log
2022-02-18 11:48:35,835 INFO    MainThread:11342 [wandb_init.py:init():404] calling init triggers
2022-02-18 11:48:35,835 INFO    MainThread:11342 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-02-18 11:48:35,835 INFO    MainThread:11342 [wandb_init.py:init():426] re-initializing run, found existing run on stack: 1dwjg9ha
2022-02-18 11:48:35,837 INFO    MainThread:11342 [wandb_run.py:finish():1457] finishing run simon-donike/SRCNN/1dwjg9ha
2022-02-18 11:48:35,838 INFO    MainThread:11342 [jupyter.py:save_history():429] not saving jupyter history
2022-02-18 11:48:35,838 INFO    MainThread:11342 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-18 11:48:35,839 INFO    MainThread:11342 [wandb_init.py:_jupyter_teardown():303] cleaning up jupyter logic
2022-02-18 11:48:35,839 INFO    MainThread:11342 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-02-18 11:48:35,840 INFO    MainThread:11342 [wandb_run.py:_restore():1752] restore
2022-02-18 11:48:37,029 INFO    MainThread:11342 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  total_bytes: 761
}

2022-02-18 11:48:37,302 INFO    MainThread:11342 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 761
}

2022-02-18 11:48:37,406 INFO    MainThread:11342 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 25181
}

2022-02-18 11:48:37,510 INFO    MainThread:11342 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 25181
}

2022-02-18 11:48:37,612 INFO    MainThread:11342 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25181
  total_bytes: 25181
}

2022-02-18 11:48:37,717 INFO    MainThread:11342 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25181
  total_bytes: 25181
}

2022-02-18 11:48:37,819 INFO    MainThread:11342 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25181
  total_bytes: 25181
}

2022-02-18 11:48:37,922 INFO    MainThread:11342 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25181
  total_bytes: 25181
}

2022-02-18 11:48:38,025 INFO    MainThread:11342 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25181
  total_bytes: 25181
}

2022-02-18 11:48:38,128 INFO    MainThread:11342 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25181
  total_bytes: 25181
}

2022-02-18 11:48:38,638 INFO    MainThread:11342 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25181
  total_bytes: 25181
}

2022-02-18 11:48:39,085 INFO    MainThread:11342 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25181
  total_bytes: 25181
}
local_info {
}

2022-02-18 11:48:40,895 INFO    MainThread:11342 [wandb_run.py:_append_files():2180] logging synced files
2022-02-18 11:48:40,905 INFO    MainThread:11342 [wandb_init.py:init():460] starting backend
2022-02-18 11:48:40,905 INFO    MainThread:11342 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-18 11:48:40,912 INFO    MainThread:11342 [backend.py:ensure_launched():216] starting backend process...
2022-02-18 11:48:41,087 INFO    MainThread:11342 [backend.py:ensure_launched():222] started backend process with pid: 11510
2022-02-18 11:48:41,089 INFO    MainThread:11342 [wandb_init.py:init():469] backend started and connected
2022-02-18 11:48:41,119 INFO    MainThread:11342 [wandb_run.py:_label_probe_notebook():911] probe notebook
2022-02-18 11:48:41,119 INFO    MainThread:11342 [wandb_run.py:_label_probe_notebook():921] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-02-18 11:48:41,120 INFO    MainThread:11342 [wandb_init.py:init():533] updated telemetry
2022-02-18 11:48:41,772 INFO    MainThread:11342 [wandb_init.py:init():563] communicating current version
2022-02-18 11:48:43,209 INFO    MainThread:11342 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-18 11:48:43,211 INFO    MainThread:11342 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-18 11:48:43,426 INFO    MainThread:11342 [wandb_init.py:init():606] starting run threads in backend
2022-02-18 11:48:48,440 INFO    MainThread:11342 [wandb_run.py:_console_start():1810] atexit reg
2022-02-18 11:48:48,441 INFO    MainThread:11342 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-02-18 11:48:48,441 INFO    MainThread:11342 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-02-18 11:48:48,442 INFO    MainThread:11342 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-18 11:48:48,442 INFO    MainThread:11342 [wandb_init.py:init():633] run started, returning control to user process
2022-02-18 11:59:27,616 INFO    MainThread:11342 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-02-18 11:59:27,618 INFO    MainThread:11342 [wandb_run.py:_restore():1752] restore
2022-02-18 11:59:28,426 INFO    MainThread:11342 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 761
}

2022-02-18 11:59:28,652 INFO    MainThread:11342 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 761
}

2022-02-18 11:59:28,830 INFO    MainThread:11342 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 10079
}

2022-02-18 11:59:28,934 INFO    MainThread:11342 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 25271
}

2022-02-18 11:59:29,051 INFO    MainThread:11342 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 25271
}

2022-02-18 11:59:29,154 INFO    MainThread:11342 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 25271
  total_bytes: 25271
}

2022-02-18 11:59:29,258 INFO    MainThread:11342 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 25271
  total_bytes: 25271
}

2022-02-18 11:59:29,364 INFO    MainThread:11342 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 25271
  total_bytes: 25271
}

2022-02-18 11:59:29,467 INFO    MainThread:11342 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 25271
  total_bytes: 25271
}

2022-02-18 11:59:29,569 INFO    MainThread:11342 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 25271
  total_bytes: 25271
}

2022-02-18 11:59:29,671 INFO    MainThread:11342 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 25271
  total_bytes: 25271
}

